earth_enterprise/src/fusion/gepublish/PublisherClient.cpp

```diff
@@ -1271,36 +1271,13 @@ bool PublisherClient::UploadFiles(ServerType server_type,
   if (is_server_host_same_as_publishing)  {
     notify(NFY_DEBUG, "Transfering files locally");
     size_t num_entries = entries.size();
-    std::vector<ManifestEntry> dependent_entries;
-    size_t i = 0;
-    for (; i < num_entries; ++i) {
+    for (size_t i = 0; i < num_entries; ++i) {
       const ManifestEntry& entry = entries[i];
 
       if (!LocalTransferWithRetry(server_prefix, host_root, server_type,
-                                  tmpdir, entry.current_path, entry.orig_path))
+                                  tmpdir, entry.current_path, entry.orig_path)) {
         return false;
-
-      for(size_t iDep = 0; iDep < entry.dependents.size(); ++iDep) {
```

Comment:
Any reason all of this was removed? Is it not necessary?

Reply:
No it is not...  I got confused and this was double adding files.  It 'worked' but it re-copied the files twice.  If you are doing links it probably would not be a huge performance issue but if are are actually doing a copy it could be.  Your comment about the strange percent made be go back re-look at things and that is when I discovered the error.